sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2834,24 +2834,24 @@
         }
       },
       "CloudToDeviceMethod": {
-        "description": "Parameters to execute a direct method on the device",
+        "description": "Parameters to execute a direct method on the device.",
         "type": "object",
         "properties": {
           "methodName": {
-            "description": "Method to run",
+            "description": "Name of the method which is to be executed.",
```

Comment:
Name of the method to be executed?

Reply:
Was this changed at all? I think "which is" makes the sentence roundabout. You can just say "Name of the method to be executed on the device"

Reply:
It now says `Name of the method to be executed.`. You are possibly looking at an outdated commit.